framework/source/class/qx/ui/core/MExecutable.js

```diff
@@ -82,7 +82,6 @@ qx.Mixin.define("qx.ui.core.MExecutable",
      */
     _bindableProperties :
     [
-      "enabled",
```

Comment:
Isnt the principal of `MExecutable` that it's added to a `Widget` that needs to pull in changes to `enabled` and other properties from the `Command`?  In which case, it's acceptable for the Widget to bind it's enabled flag or any other property to the Widget it's included into - it's just a design decision.

The problem with enabled is a more general one, IMHO, where different parts of the code want to set enabled and have the values merged in some meaningful way.  I have this problem when defining a UI that enables and disables other parts of the UI based on state (eg a `CheckBox` is off, so therefore a `GroupBox` of fields is not editable and has `enabled` set to false); but then there are also user permissions, and if the current user is not allowed to edit any of the fields everything is disabled but the CheckBox is on so the code wants to enable the fields in the GroupBox....

It sounds like @cajus' problem is the same thing, MExecutable's signalling or enabled is conflicting with the enabled setting of other code; but removing it altogether is not the right answer either because there are good reasons for reflecting Command properties in Widget, which is half the reason MExecutable exists in the first place.